collector/benchmarks/README.md

```diff
@@ -35,7 +35,7 @@ They mostly consist of real-world crates.
   compiler in certain ways.
 - **piston-image**: A modular game engine. An interesting Rust program.
 - **regex**: A regular expression parser. Used by many Rust programs.
-- **ripgrep**: A line-oriented search tool. A widely-used utility.
+- **ripgrep-13.0.0**: A line-oriented search tool. A widely-used utility.
```

Comment:
Strictly speaking, you should leave the old `ripgrep` there until it's removed in a later PR.